packages/docs/locales/en/en.json

```diff
@@ -20,7 +21,14 @@
         "stateful": "Add possibility to work with component without setting `v-model`",
         "color": "Color of the component (theme string or *HEX* string)",
         "textColor": "Text color (theme string or *HEX* string)",
-        "rules": "Validation rules <!-- TODO Add link -->"
+        "rules": "Validation rules <!-- TODO Add link -->",
+        "to": "`router-link` attribute.",
```

Comment:
Needs more info, see https://vuetifyjs.com/en/components/buttons/

Reply:
ok, do we need link to vue-router docs?